src/api/v1/paths/pizzas/{pizzaId}.js

```diff
@@ -9,20 +10,45 @@ import { errorBuilder, errorHandler } from 'errors/errors';
 const get = async (req, res) => {
   try {
     const result = await getPizzaById(req.params.pizzaId, req.query);
-    if (result === null) {
-      return errorBuilder(res, '404', `No pizza with ID ${req.params.pizzaId} found`);
-    }
     return res.send(result);
   } catch (err) {
+    if (err instanceof ResourceNotFoundError) {
+      return errorBuilder(res, '404', `No pizza with ID ${err.id} found`);
+    }
     return errorHandler(res, err);
   }
 };
 
 /**
- * stub
- * @returns null
+ * Update a pizza by ID
+ *
+ * @type {RequestHandler}
  */
-const patch = () => null;
+const patch = async (req, res) => {
+  try {
+    if (req.query.pizzaId !== req.body.data.id) {
+      return errorBuilder(
+        res,
+        '409',
+        `ID ${req.params.pizzaId} in URL does not match ID ${req.body.data.id}`,
+      );
+    }
```

Comment:
I seem to be getting this message even when the two IDs are the same: `"detail": "ID 1 in URL does not match ID 1"`